exp_build/plugin.js

```diff
@@ -99,12 +99,28 @@ class Plugin {
       shell.exec("mkdir plugins");
     }
     pluginConfig.map((plugin) => {
+      if(plugin.label == 'Code Assessment') {
```

Comment:
Please remove this code. If each plugin introduces logic specific to itself in the build framework then this model would not scale. Plugins should be processed using generic code and specific functionality should depend on the configuration parameters.

Reply:
done